src/ios/SocialSharing.m

```diff
@@ -735,8 +735,20 @@ -(NSURL*)getFile: (NSString *)fileName {
     if ([fileName hasPrefix:@"http"]) {
       NSURL *url = [NSURL URLWithString:fileName];
       NSData *fileData = [NSData dataWithContentsOfURL:url];
-      NSString *name = (NSString*)[[fileName componentsSeparatedByString: @"/"] lastObject];
-      file = [NSURL fileURLWithPath:[self storeInFile:[name componentsSeparatedByString: @"?"][0] fileData:fileData]];
+      NSURLRequest *request = [NSURLRequest requestWithURL: url];
+      NSHTTPURLResponse *response;
+      [NSURLConnection sendSynchronousRequest: request returningResponse: &response error: nil];
+      if ([response respondsToSelector:@selector(allHeaderFields)]) {
```

Comment:
What about the case where this is false? (if that's even possible)

Reply:
Added it